src/internal/m365/collection/site/backup.go

```diff
@@ -159,6 +159,7 @@ func CollectLists(
 	var (
 		el        = errs.Local()
 		spcs      = make([]data.BackupCollection, 0)
+		spcsMap   = make(map[string]data.BackupCollection)
```

Comment:
Could you add `// list id -> backup collection` or similar as a comment on top?

Reply:
let me make the change in the next PR in the chain